Cargo.toml

```diff
@@ -75,4 +75,4 @@ name = "assets"
 path = "examples/05_assets/main.rs"
 
 [workspace]
-members = ["src/renderer"]
+members = ["assets", "src/renderer"]
```

Comment:
I would prefer to name this folder `amethyst_assets` simply to differentiate subcrates from the `book` and `src` directories. It may look a little verbose, but at least users won't be misled into thinking there are real game assets in that folder, as @lschmierer [pointed out in the Gitter chat](https://gitter.im/amethyst/engine?at=5969031689aea4761d9b76c3) a while ago.